javascript.html.markdown

```diff
@@ -613,6 +613,12 @@ const isEven = (number) => {
 
 isEven(7); // false
 
+// An even shorter way to write a function similar to the above one is:
```

Comment:
hey! feel free to delete the original isEven function (and adjust your comment accordingly). Your version is superior